pkg/iam/interfaces.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// iam package provides interfaces around the google resourcemanager
+// organizations, folders, projects client to handles IAM policy update.
+package iam
```

Comment:
Moved the interface into this new package, if it stays in the handler package, I cannot depend on the testutil since testutil also uses this interface (import cycle). Is this the proper way to do?

Reply:
I believe if you decouple `IAMHandler` from the command, you wouldn't need this change.